orion/pkg/repos/migrations/000005_create_table_locations.up.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE locations (
+	id int unsigned NOT NULL AUTO_INCREMENT,
+	created_at datetime NOT NULL,
+	updated_at datetime NOT NULL,
+	deleted_at datetime,
+	loc_id varchar(64) NOT NULL,
+	street varchar(255) NOT NULL,
+	city varchar(64) NOT NULL,
+	state varchar(64) NOT NULL,
+	zipcode varchar(64) NOT NULL,
+	room varchar(64) NOT NULL,
```

Comment:
On second thought, let's turn this into a Nullable (get rid of `NOT NULL`).
If we give a home address, it probably doesn't make sense to have a "room" number.